packages/tools/src/utilities/cine/playClip.ts

```diff
@@ -37,20 +45,23 @@ function playClip(
     );
   }
 
-  const { viewport } = enabledElement;
+  // 4D Cine is enabled by default
+  playClipOptions.dynamicCineEnabled =
+    playClipOptions.dynamicCineEnabled ?? true;
```

Comment:
it should be default false, 4D cine is maybe 5% of use cases, even less

Reply:
if `dynamicCineEnabled` but it is a 3D volume nothing will change. It will work as before. But if it is a 4D volume it will always scroll through time.